core/tooltip.js

```diff
@@ -4,216 +4,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/**
- * @fileoverview Library to create tooltips for Blockly.
- * First, call init() after onload.
- * Second, set the 'tooltip' property on any SVG element that needs a tooltip.
- * If the tooltip is a string, then that message will be displayed.
- * If the tooltip is an SVG element, then that object's tooltip will be used.
- * Third, call bindMouseEvents(e) passing the SVG element.
- */
 'use strict';
 
-/**
- * Library to create tooltips for Blockly.
- * First, call init() after onload.
- * Second, set the 'tooltip' property on any SVG element that needs a tooltip.
- * If the tooltip is a string, then that message will be displayed.
- * If the tooltip is an SVG element, then that object's tooltip will be used.
- * Third, call bindMouseEvents(e) passing the SVG element.
- * @namespace Blockly.Tooltip
- */
 goog.module('Blockly.Tooltip');
 
 const blocklyString = goog.require('Blockly.utils.string');
 const browserEvents = goog.require('Blockly.browserEvents');
 const common = goog.require('Blockly.common');
-const deprecation = goog.require('Blockly.utils.deprecation');
-
-
-/**
- * A type which can define a tooltip.
- * Either a string, an object containing a tooltip property, or a function which
- * returns either a string, or another arbitrarily nested function which
- * eventually unwinds to a string.
- * @typedef {string|{tooltip}|function(): (string|!Function)}
- * @alias Blockly.Tooltip.TipInfo
- */
-let TipInfo;
-exports.TipInfo = TipInfo;
-
-/**
- * Is a tooltip currently showing?
- * @type {boolean}
- */
-let visible = false;
-
-/**
- * Returns whether or not a tooltip is showing
- * @returns {boolean} True if a tooltip is showing
- * @alias Blockly.Tooltip.isVisible
- */
-const isVisible = function() {
-  return visible;
-};
-exports.isVisible = isVisible;
-
-Object.defineProperties(exports, {
-  /**
-   * Is a tooltip currently showing?
-   * @name Blockly.Tooltip.visible
-   * @type {boolean}
-   * @deprecated Use Blockly.Tooltip.isVisible() instead.  (September
-   *     2021)
-   * @suppress {checkTypes}
-   */
-  visible: {
```

Comment:
If we don't want to have this, could we add an `isVisible` method on to this class and then mention it as a breaking change in the PR comment? It isn't the best since we are making a public API without any proof/use case in mind, but since it was already public seems like it would be nice to offer a workaround.